src/osd/OSDMap.cc

```diff
@@ -1022,16 +1022,16 @@ int OSDMap::calc_num_osds()
   return num_osd;
 }
 
-void OSDMap::count_full_nearfull_osds(int *full, int *nearfull) const
+void OSDMap::count_full_nearfull_osds(int& full, int& nearfull) const
```

Comment:
i agree on 1 and 2. anyway, IMHO, just like the code style, we'd better stick with it for consistency unless we have some overwhelming reason to change it.